### PR TITLE
Scan site for non-inclusive language

### PR DESCRIPTION
**Pull request by bonnieAcameron**
## Summary

Basic description of work done.

## Preview

[Link to Preview]()

## Solution

Provide a summary of the solution this PR offers.

## How To Test

1. First Step
2. Second Step
3. Third Step

### CODE REVIEW

File: content/news/2016/09/2016-09-12-dnssec-vs-elastic-load-balancers-the-zone-apex-problem.md
Comment (ToniBonittoGSA) [inline content/news/2016/09/2016-09-12-dnssec-vs-elastic-load-balancers-the-zone-apex-problem.md:1]: > Then, every minute, “DNS primary acting as

 should be:

> Then, every minute, “DNS secondaries" acting as ...

Original:

> Then, every minute, “DNS slaves” acting as ...

File: content/news/2020/12/2020-12-21-deceptive-design-how-to-identify-and-combat-consequence-design.md
Comment (ToniBonittoGSA) [inline content/news/2020/12/2020-12-21-deceptive-design-how-to-identify-and-combat-consequence-design.md:1]: The alternate for "dark patterns" is "deceptive patterns"

https://www.nngroup.com/articles/deceptive-patterns/

File: content/resources/deceptive-design-how-to-identify-and-combat-consequence-design.md
Comment (ToniBonittoGSA) [inline content/resources/deceptive-design-how-to-identify-and-combat-consequence-design.md:1]: Original
> Deceptive techniques like “dark patterns” and “hostile design” trick people into ...

edit
> Deceptive techniques like “hostile design” trick people into ...

change to
> Techniques like “deceptive patterns” and “hostile design” trick people into ...

File: content/news/2013/08/2013-08-13-trends-on-tuesday-people-are-excited-about-mobile.md
Comment (ToniBonittoGSA) [inline content/news/2013/08/2013-08-13-trends-on-tuesday-people-are-excited-about-mobile.md:1]: Are there any pages linking to the original URL `https://digital.gov/2013/08/13/trends-on-tuesday-people-are-crazy-about-mobile/` or the .md file `{{< link "trends-on-tuesday-people-are-crazy-about-mobile.md" >}}` ?

If so, add an `alias` for `trends-on-tuesday-people-are-crazy-about-mobile` in the front matter

File: content/news/2016/07/2016-07-12-trends-on-tuesday-adoption-for-pokemon-go-mobile-app-surges.md
Comment (ToniBonittoGSA) [inline content/news/2016/07/2016-07-12-trends-on-tuesday-adoption-for-pokemon-go-mobile-app-surges.md:1]: Are there any pages linking to the original URL `https://digital.gov/2016/07/12/trends-on-tuesday-adoption-for-pokemon-go-mobile-app-goes-crazy/` or the .md filename `{{< link "trends-on-tuesday-adoption-for-pokemon-go-mobile-app-goes-crazy.md" >}}` ?

If so, add an `alias` for `trends-on-tuesday-adoption-for-pokemon-go-mobile-app-goes-crazy` in the front matter

**Review by camillemorrow — APPROVED** [review]

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**